cirq-core/cirq/ops/measure_util_test.py

```diff
@@ -28,6 +28,8 @@ def test_measure_qubits():
 
     assert cirq.measure(a) == cirq.MeasurementGate(num_qubits=1, key='a').on(a)
     assert cirq.measure(a, b) == cirq.MeasurementGate(num_qubits=2, key='a,b').on(a, b)
+    assert cirq.measure([a, b]) == [cirq.MeasurementGate(num_qubits=1, key='a').on(a),
+                                    cirq.MeasurementGate(num_qubits=1, key='b').on(b)]
```

Comment:
This behavior seems highly unintuitive to me. I would think we'd want `cirq.measure(a, b)` and `cirq.measure([a, b])` to be treated exactly the same, rather than having the latter expand into separate measurement gates. This is similar to `on_each` itself where `gate.on_each(a, b)` and `gate.on_each([a, b])` do the same thing. In principle I'm ok with modifying the `measure` function to accept qids as varargs or an iterable, but I think it should produce one measurement gate in either case. SImilarly, the `measure_each` function could be modified to accepts qids as varargs or an iterable and should produce separate 1-qubit measurement gates in either case.

Reply:
I see. I think I'll have to add another function to keep the `measure` function dry. I can take a shot at the `measure_each` case as well. Might all be up tonight, but `measure` should be at least.